src/serial.rs

```diff
@@ -240,6 +247,8 @@ macro_rules! hal {
                     usart.cr2.reset();
                     usart.cr3.reset();
 
+                    usart.cr1.modify(|_, w| w.fifoen().set_bit());
```

Comment:
Same as before: should this be enabled unconditionally? How does enabling it affect behavior of existing implementations?